testsrc/by/javatr/model/BallTest.java

```diff
@@ -47,7 +53,14 @@ public void equalsTest() {
     public void equalsBothNullColorsTest() {
         Ball ball1 = new Ball(null, 2.5);
         Ball ball2 = new Ball(null, 2.5);
-        Assert.assertEquals(ball1, ball2);
+        Assert.assertEquals(ball1,ball2);
+    }
+
+    @Test (expected = InvalidColorException.class)
```

Comment:
Why do you expect InvalidColorException? Your test should either expect an exception and assert some condition such as assertEquals, but not both.